nss-day-cohort-44/nutshell-whiskered-gnomes
# Ks public chat

# Description
- adds getMessages to fetch data from messages resource
- adds useMessages to sort messages by timestamp and save them in a usable array
- adds messageHTML to return html for a message object
- adds renderMessages to iterate through the messages array and call messageHTML for each object in the array
- adds MessageList to call getMessages, useMessages and renderMessages

Fixes #48 

## Type of change
Please delete options that are not relevant.

# Testing Instructions
Add and commit changes to your current branch
git checkout master
git fetch --all
git checkout ks-public-chat
serve
See that any messages you have stored in your api are rendered to the dom. Should be in timestamp order, with newest message showing at bottom. Each should have a delete button. Each username should look like a link. You might need to add a couple of message objects to your api to test this. 

# Checklist:
